sljit_src/sljitExecAllocator.c

```diff
@@ -192,6 +192,63 @@ static SLJIT_INLINE void free_chunk(void *chunk, sljit_uw size)
 
 #endif
 
+#if defined(__APPLE__) && defined(__arm64__)
+__thread jit_protect_mode arm_current_jit_protect_mode = JPM_ENABLED;
+
+inline void sljit_arm_jit_write_protect_enable()
+{
+	if (__builtin_available(macOS 11, *)) {
```

Comment:
this is likely to be also needed for some versions of iOS (and friends), probably even older than macOS 11, since it would seem this "feature" was first implemented there and its hardware support might be even 2 generations old

Reply:
JIT execution on iOS / iPadOS is explicitly forbidden. We could revisit this if that changes in the future

Reply:
make sense, there is still the open issue if an ifdef might be required to avoid building failures in iOS (and friends).  is building for iOS something that you could consider testing?

Reply:
FWIW this syntax broke my build in macOS 10.13 (using Xcode) but the alternative used in b0d7c3581f3fbcac797070423d7b15ef237f0a3c should work if I guessed correctly the version number you should have in your headers.